main.cpp

```diff
@@ -59,9 +66,44 @@ int main()
 	imshow("Display", I);
 	waitKey(0);
 	*/
+	//Just for test
```

Comment:
Better encapsulate the following code into a ExperimentData method since it would help us maintain/improve this code in the future.